CAPE/CAPE/run_cape_offline.cpp

```diff
@@ -97,19 +95,25 @@ void organizePointCloudByCell(Eigen::MatrixXf & cloud_in, Eigen::MatrixXf & clou
 
 int main(int argc, char ** argv){
 
-    string sequence;
+    bool show_visualization=false;
+    stringstream string_buff;
+
     int PATCH_SIZE;
-    if (argc>1){
+    if (argc>2){
```

Comment:
What is the command parsed here? Nothing about it in README?

Reply:
Here I did visualization optionally. I added information in README but then you told me that if we work with ENTRYPOINT then no needed part about parameters and so on. That's why I deleted it from README

Reply:
Now I added these parameters with examples in docker run command, but I think if I showed it without docker run command it might be more understandable.